promgen/tests/examples/rest.farm.1.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": 1,
+  "url": "http://promgen.example.com/farm/1",
+  "name": "test-farm",
+  "source": "promgen",
+  "hosts": [
+    {
+      "name": "host.example.com",
+      "url": "http://promgen.example.com/host/host.example.com"
+    }
+  ]
+}
```

Comment:
This file is missing an new-line character at the end:
![image](https://github.com/user-attachments/assets/a1d91725-54c2-4885-9ec9-1caf2bf9aad1)

Please add one.

Reply:
I've seen you addressed this comment and [the other one](https://github.com/line/promgen/pull/554#discussion_r1849437618) on [81e1b0d](https://github.com/line/promgen/pull/554/commits/81e1b0d99125425cf2060831eea42d53d602adb0).

Thank you!